pkg/web/api/health_checker_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"context"
+	"testing"
+
+	"github.com/conduitio/conduit/pkg/foundation/cerrors"
+	"github.com/conduitio/conduit/pkg/foundation/log"
+	"github.com/matryer/is"
+	gh "google.golang.org/grpc/health/grpc_health_v1"
+)
+
+type testChecker struct {
+	err error
+}
+
+func (t *testChecker) Check(ctx context.Context) error {
+	return t.err
+}
+
+func TestHealthChecker_Check_OK(t *testing.T) {
```

Comment:
Seems like a leftover from renaming (same for other tests).
```suggestion
func TestHealthServer_Check_OK(t *testing.T) {
```